### PR TITLE
Feature/disconnected

### PR DESCRIPTION
**Pull request by PorlyBe**
image

### CODE REVIEW

File: src/App.tsx [hunk 1]
```diff
@@ -1,80 +1,62 @@
-import React, { FC } from "react";
+import React, { FC, useEffect, useTransition } from "react";
```
Comment (micahpatng) [inline src/App.tsx:1]: `useTransition` is not being usesd

File: src/components/Footer.tsx [hunk 2]
```diff
@@ -18,14 +24,11 @@ const Wrapper = styled.div`
...
   return (
-    <>
-      <StyledFooter>
-        <Wrapper>
-          <div>Copyright 2023</div>
-          <div>Made by: Micah, Paul, Ricky</div>
-        </Wrapper>
-      </StyledFooter>
-    </>
+    <StyledFooter>
```
Comment (micahpatng) [inline src/components/Footer.tsx:27]: did the designers want this?

File: src/context/RootProvider.tsx [hunk 3]
```diff
@@ -0,0 +1,86 @@
...
+const wagmiConfig = createConfig({
+  autoConnect: true,
+  connectors: [
+    new WalletConnectConnector({
+      chains,
+      options: {
+        projectId,
+        showQrModal: false,
+        metadata,
+        // relayUrl: "wss://walletconnect-v2.ops.bitcoin.com",
```
Comment (micahpatng) [inline src/context/RootProvider.tsx:41]: we can remove?

**Comment by micahpatng** [comment]
also loading is different from design? 
Screenshot 2023-10-20 at 13 52 23

File: src/context/RootProvider.tsx (hunk 3 above)
Comment (PorlyBe) [reply src/context/RootProvider.tsx:41]: wanna keep incase they get it working

File: src/components/Footer.tsx (hunk 2 above)
Comment (PorlyBe) [reply src/components/Footer.tsx:27]: bert is coming up with a universal footer design... so not exactly like this but yeah...

File: src/App.tsx (hunk 1 above)
Comment (PorlyBe) [reply src/App.tsx]: left over from failed experiment

**Review by micahpatng — APPROVED** [review]

**Pull request merged**